globalcombined.lua

```diff
@@ -26,13 +42,46 @@ version = "0.3d"
 -- TY McSteve for reporting this to us.
 
 function et_ClientCommand(cno,cmd)
-	cmd = string.lower(cmd)
-if cmd == "forcetapout" then ---forcetapout bugfix
-	 if et.gentity_get(cno, "r.contents") == 0 then  --contents 0 =nobody
+	local cmd = string.lower(cmd)
+	local arg1 = string.lower(et.trap_Argv(1)) 
+	local byte = string.byte(arg1,1)
+	
+	if (cmd == "say" or cmd == "say_team" or cmd == "say_buddy" or cmd == "say_teamnl") and serverpassword == "" then

```

Comment:
Micha (original author) informs me this section was used purely for debugging and **should** be remove